lib/deps.nix

```diff
@@ -33,6 +33,9 @@ let
       #torch.cxxdev
     ];
     "cutlass_sycl" = [ torch.xpuPackages.cutlass-sycl ];
+    "metal-cpp" = lib.optionals pkgs.stdenv.hostPlatform.isDarwin [
```

Comment:
I think the darwin conditional is not needed here, it only gets evaluated when somebody adds the dependency.

Reply:
good catch, removed in latest